xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-preferences/xwiki-platform-notifications-preferences-api/src/main/java/org/xwiki/notifications/preferences/script/NotificationPreferenceScriptService.java

```diff
@@ -292,7 +297,26 @@ public NotificationEmailDiffType getDiffType()
      */
     public NotificationEmailDiffType getDiffType(String userId)
     {
-        return emailUserPreferenceManager.getDiffType(userId);
+        return emailUserPreferenceManager.getDiffType(userReferenceResolver.resolve(userId));
+    }
+
+    /**
+     * @return the email notification interval configured for the current user
+     * @since 14.4.2
```

Comment:
You forgot to change the since. And also missing Unstable there.

Reply:
thx done